### PR TITLE
Add generation of  meta data to generate_eopatch

### PR DESCRIPTION
**Pull request by jgersak**

### CODE REVIEW

File: core/eolearn/core/utils/testing.py [hunk 1]
```diff
@@ -50,7 +52,7 @@ def generate_eopatch(
 ) -> EOPatch:
     """A class for generating EOPatches with dummy data."""
     config = config if config is not None else PatchGeneratorConfig()
-    supported_feature_types = [ftype for ftype in FeatureType if ftype.is_array()]
+    supported_feature_types = [ftype for ftype in FeatureType if ftype.is_array()].append(FeatureType.META_INFO)
```
Comment (zigaLuksic) [inline core/eolearn/core/utils/testing.py:55]: this is a clear mistake. Investigate why our tests did not catch it

Comment (zigaLuksic) [reply core/eolearn/core/utils/testing.py:55]: (`mypy` did, good boy :dog2: )

Comment (jgersak) [reply core/eolearn/core/utils/testing.py]: > Investigate why our tests did not catch it 

supported_feature_types went to `None` so all is "supported" we do not have test for that

Comment (zigaLuksic) [reply core/eolearn/core/utils/testing.py]: if all is supported, why didnt our tests for "vector features are not allowed" detect that?

Comment (jgersak) [reply core/eolearn/core/utils/testing.py]: ValueError come from wrong dimension assigned to feature

File: core/eolearn/core/utils/testing.py [hunk 2]
```diff
@@ -60,6 +63,9 @@ def generate_eopatch(
     # fill eopatch with random data
     # note: the patch generation functionality could be extended by generating extra random features
     for ftype, fname in parsed_features:
+        if ftype == FeatureType.META_INFO:
+            patch[(ftype, fname)] = "".join(random.choices(string.ascii_letters, k=20))
```
Comment (zigaLuksic) [inline core/eolearn/core/utils/testing.py:67]: use the `rng` that you have, or things are not repeatable.
Include a meta feature into the test where you ensure that seeds work, it should detect that

File: core/eolearn/core/utils/testing.py [hunk 3]
```diff
@@ -51,6 +53,7 @@ def generate_eopatch(
     """A class for generating EOPatches with dummy data."""
     config = config if config is not None else PatchGeneratorConfig()
     supported_feature_types = [ftype for ftype in FeatureType if ftype.is_array()]
+    supported_feature_types.append(FeatureType.META_INFO)
     parsed_features = FeatureParser(features or [], supported_feature_types).get_features()
```
Comment (zigaLuksic) [inline core/eolearn/core/utils/testing.py:57]: try using a callable for that. Probably `lambda fty: fty.is_array() or fty == FeatureType.META_INFO`?

File: core/eolearn/core/utils/testing.py (hunk 1 above)
Comment (jgersak) [reply core/eolearn/core/utils/testing.py]: Change error to `NotImplementedError`

File: core/eolearn/core/utils/testing.py (hunk 3 above)
Comment (jgersak) [reply core/eolearn/core/utils/testing.py]: done

**Review by zigaLuksic — APPROVED** [review]

**Pull request merged**